hod/local.py

```diff
@@ -76,7 +79,7 @@ def known_cluster_labels():
     """
     Return list of known cluster labels.
     """
-    path = cluster_config_dir()
+    path = cluster_info_dir()
     if os.path.exists(path):
```

Comment:
actually, I think we should keep it this way, since otherwise we can't discriminate between a directory that doesn't exist (which is fine, no cluster info available), and a filesystem issue (bad)